### DISCUSSION TITLE
GitHub Actions deprecation may affect this action

### DISCUSSION THREAD

**Discussion by emmahsax** [JamesIves/github-pages-deploy-action#462] [Q&A] [answered]
Hello @JamesIves,

GitHub announced a new deprecation recently, described [here](https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/). They want GitHub Action authors to update any usage of `@actions/core` to be `v1.2.6` or higher to receive new `addPath` and `exportVariable` functions. I see [this line](https://github.com/JamesIves/github-pages-deploy-action/blob/dev/package.json#L36) in the `package.json`, and just want to verify that this means that this action is already using the required version.

Thank you in advance.

Emma

**Comment by JamesIves** [comment, marked as answer]
Thanks Emma!

Yep things should be all up to date! There was a dependency for the ssh functionality mentioned in the readme that used a previous version of this which was updated here: https://github.com/JamesIves/github-pages-deploy-action/issues/454 

Plans have been made to natively support ssh without relying on a dependency as a result but it won't be ready for a few more weeks at the earliest: https://github.com/JamesIves/github-pages-deploy-action/issues/459

**Comment by JamesIves** [comment]
Moved to discussions as this is a great example of a Q&A! 😄
